s3prl/upstream/mos_prediction/README.md

```diff
@@ -0,0 +1,33 @@
+# MOS Prediction
+
+Official Implementation of "Utilizing Self-supervised Representations for MOS Prediction", will be presented at INTERSPEECH 2021 [[arXiv](https://arxiv.org/abs/2104.03017)]
+
+This code provides a automatic Mean Opinion Score (MOS) Predictor that utilize the self-supervised representations, implementing in PyTorch. The MOS Predictor is serves as an upstream model in s3prl toolkit.
+
+## Example Usage
+There are currently three available checkpoints that used different self-supervised speech models, namely: `mos_wav2vec2`, `mos_tera`, `mos_apc`.
+
+And here's the example usage:
+```python
+import torch
+
+device = 'cuda'
+mos_predictor = torch.hub.load('s3prl/s3prl', 'mos_wav2vec2').to(device)
```

Comment:
Hey I just catch another `torch.hub.load` usage. I would like to replace with `from s3prl.hub import` usage.